package-lock.json

```diff
@@ -1,4139 +1,8 @@
 {
   "name": "@progresskinvey/saml2js",
   "version": "0.2.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Is there a particular reason for the version change/downgrade?
I see a lot of changes in the lockfile.

Reply:
I have reverted the change